<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoForm</class>
 <widget class="QWidget" name="InfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="numOfPlayersLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of players</string>
   </property>
  </widget>
  <widget class="QLabel" name="numOfCardsLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of pairs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QComboBox" name="playersNameBox">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>160</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="numOfPlayersSpinBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>43</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="numOfCardsSpinBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>43</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="playerNamelabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Type the name of player in the box below</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
